build/templates/session.py.mako

```diff
@@ -48,7 +48,23 @@ class ${session_context_manager}(object):
 
 
 % endif
-class _SessionBase(object):
+% for rep_cap in config['repeated_capabilities']:
+class _${rep_cap['python_class_name']}(object):
+    def __init__(self, ${config['session_handle_parameter_name']}):
+        self._${config['session_handle_parameter_name']} = ${config['session_handle_parameter_name']}
+
+    def __getitem__(self, repeated_capability):
+        '''Set/get properties or call methods with a repeated capability (i.e. channels)'''
+        if isinstance(repeated_capability, list):
+            rep_cap_list = [str(r) if str(r).startswith('${rep_cap['prefix']}') else '${rep_cap['prefix']}' + str(r) for r in repeated_capability]
```

Comment:
Hmmm... IVI repeated capability strings are case insensitive, but the prefix check here is case sensitive.

Reply:
Made check case insensitive.